3p/frame.max.html

```diff
@@ -7,7 +7,9 @@
 try {
   url = JSON.parse(window.name).bootstrap;
 } catch (e) {}
-document.write('<script src="' + url + '"><' + '/script>');
+var script = document.createElement('script');
+script.src = url;
```

Comment:
if we are trying to make this more "secure" would adding trusted types (if supported) help?

Reply:
I thought about it. The list of URLs here will need to be very permissible. It needs to allow many 3p domains that hosts the JS which AMP has no control of. We will certainly not be worse off but the benefits are not huge. Is there a mandate for it?

Reply:
there is from security, but lets skip it for this file for now since i dont know if it applies

Reply:
LGTM